lib/velum/salt.rb

```diff
@@ -41,6 +41,26 @@ def self.minions
       JSON.parse(res.body)["return"].first
     end
 
+    # Returns the minions that have not been accepted into the cluster
+    def self.pending_minions
+      res = perform_request(endpoint: "/", method: "post",
+                            data: { client: "wheel",
+                                    fun:    "key.list" ,
+                                    match:  "all",
+                                    tgt:    "ca" })
+      JSON.parse(res.body)["return"].first["data"]["return"]["minions_pre"]
```

Comment:
Yeap - this was a quick push to get a quick round of feedback on the concept itself. Will add it now